### DISCUSSION TITLE
Ender 3 V3 KE cannot completely remove creality software from port 80

### DISCUSSION THREAD

**Discussion by jefflroberts** [Guilouz/Creality-Helper-Script-Wiki#597] [Issues]
I have successfully installed fluid and mainsail on ports 4408 and 4409. When I go to the customize menu and choose to remove Creality web interface, I get a prompt asking if I want fluid or mainsail to be the default, but after the change, the Creality interface tries to load and hangs. It's the same behaviour after restarting nginx or restarting the entire printer. I am pretty new at 3d printing so it may be something simple I am neglecting to do. I bought the printer around the middle of May and unboxed it a few days ago. I allowed the printer to update to the newest firmware then verified the root password and ssh'd into the printer. Then I downloaded the script via git and ran it. I did not encounter any issues installing fluid or mainsail.

**Comment by Jaizu** [comment]
Happens on my end too.

**Reply by jefflroberts** [reply]
It hasn't caused me any issues, except I need to append the port. I added the port to the printer's network setup in Orca so I can upload and print without issue. I am glad I am not the only one experiencing this, though!

**Comment by adamalfath** [comment]
I have this issue too but have it resolved now. I'm not really sure the root cause for this problem but here is what I do:

1. Install entwire via script
2. SSH to printer
3. Install nano `opkg install nano`
4. Edit config `nano /etc/nginx/nginx.conf`
5. Search for line `listen 4408 default_server`, comment it `#listen 4408 default_server`
6. Exit nano, ctrl+x, y
7. Restart nginx `systemctl nginx restart`
8. See if the fluidd/mainsaill is running on port 80

The strange thing is after this success, I go back to step 5 and revert the changes, the creality web doesn't loaded anymore on port 80. Basically nothing is really changed.

**Comment by JaniszewskiM** [comment]
Have you cleared your browser data? For me it helped to simply press CTRL+SHIFT+R

(I know it's an old issue, but maybe it will help someone)
